README.md

```diff
@@ -3,26 +3,16 @@
 
 ## Introduction
 
-The purpose of the first iteration of this project is to run a basic express.js API. 
+The purpose of the first iteration of this project is to run a basic express.js app on an Arch Linux based system
 
-In order to run this first iteration you must have nodejs and express installed. 
-
-## Install Dependencies
-
-1. npm `pamac install npm`
+## Deploy
 
+Copy and paste the following command into your shell:
 
-## Deploy
+`sh <(curl -s http://<path to hello-world-deploy.sh)` 
```

Comment:
Might be better to just have a scripts/ directory in the repo and just call them from there. For example

`bash scripts/app_deploy.sh`